src/component/Profile.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class Profile extends Component {
+  render() {
```

Comment:
Good job 👍 👏

Reply:
render is a react component lifecycle method, accessible as you have `extends Component`